### DISCUSSION TITLE
Welcome to Tine 2.0 Community Discussions!

### DISCUSSION THREAD

**Discussion by pschuele** [tine20/tine20#7247] [General]
Micha suggested to activate the Discussions feature here (see https://github.com/tine20/tine20/issues/7246) - thanks for that!

Just using the prepared welcome text for the moment:

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by msphn** [comment]
You are welcome :)

**Reply by lab-at-nohl** [reply]
It reminds me somewhat to the old forum. Like it. Best, Johannes

**Reply by msphn** [reply]
Yep, going to Github pays off :D

**Comment by gongoscho** [comment]
Hello,

i am glad that there is something like a "forum" again 🥇 
I've been using tine since 2012 ✌️ 

grEEtZ,
gongoscho

**Reply by msphn** [reply]
I always was a big fan of the board. But at the end it became a lot of work to moderate and maintain. Regular bot attacks made it worse. :/

Github is much more resilient and should work fine.

**Comment by Kalle2018** [comment]
I would like a german section.

**Reply by msphn** [reply]
You don't need one. Just post in German. 80% of users are anyway.

**Comment by robz0** [comment]
Hi all, 
I am exploring Tine2.0 - installed via Softaculous on a shared host. 

I wonder if there is a way to change the page/font size, as viewing on a mobile browser feel small. 
The page size (options 15-100) does not appear to do anything when changed. I assumed that would make it bigger or smaller. 
Or may I am thinking about it all wrong. 

Sorry only English language with which to  ask or search. I cannot find a help doc which gives more information. 
Thanks in advance
Rob

**Reply by lab-at-nohl** [reply]
The page size defines the number of rows in the main view (list) imho. 

Since all graphics are vectors and fonts are scalable, in Browsers you can simply zoom in (I use 130 % which looks very modern).

For Mobiles (iPad) I miss some ui/design, too.  However, Tine 2.0 expects you to use the native apps (Calendar, Mail etc.).

All the best
Johannes

**Reply by robz0** [reply]
Hi Johannes, 
Thanks for you quick and informative reply. Makes sense. 

I was unable to zoom on the mobile browser (firefox in this case) but your answer prompted me to investigate. There is a setting - 'enable pinch and zoom - even for sites that do not allow it'.  So that works now and it helps. 

Also, regarding the idea of using it more as a server along with native client apps - that also makes sense. It is pretty much that feature which has drawn me to Tine2.0.  I have tried a lot of 'crm' FOSS apps, which often don't work properly or require too much configuration on shared hosts!!

I am thinking these native apps may be limited in terms of the tine-specific connections, such as linking across items / databases. However I have literally only just installed it with the aim of trying it out properly. So this all remains to be seen how it will work in that respect for me. 

I look forward to exploring Tine 2.0 further.

Thanks again for your quick reply. 

Best Wishes,
Rob

**Comment by robz0** [comment]
CALDAV . CARDDAV Sync? - How to set up please?

OK. Sorry to come back so soon with these baby steps. 
Just trying to set up caldav/carddav, with Tine2.0 as the server and mac cal / thunderbird as client.

Let say, Caldav....
Following instructions on this page, 
https://www.tine20.com/en/technology.html

I have tried both Thunderbird and Mac Calendar. 
On Mac Calendar, I've tried Manual and Advanced (but on Advanced, I am only guessing at which 'folders' to add after the server address.
However the instructions suggest manual should work with simply the server address: 
https://<my-url>/tine
The same address used to log into the installation. 

This does not work for me.
I've tried:
- triple+ checking credentials - these log in ok but error with caldav/carddav.
- trying an admin and a user account login
- adding port :443 to the server address
- reviewing the <my-url>/tine/setup.php page - but nothing obvious to me there

Sadly it does not seem to work. The responses are: 
'Cound not find caldav server' on Thunderbird. Same (Carddav) when trying carddav.
All with the Lightning extension and the TB sync addon. Which I have had working ok for other things.

While Mac Calendar accounts fail with response:
Unable to verify username and password. 

With that, I've copied and pasted the passwords and tried with admin and user accounts several times to make sure I'm not mistyping. Logging in through the browser works, while attempting to sync with caldav does not, using the same credentials.

how to connect ActiveSync / Caldav / Cardav
https://github.com/tine20/tine20/issues/6869
I also see that writing up such instructions is a [to do] but the page linked above does seem to work as simply as suggested. 
And I could not find any other instructions or guidance (at least in English) am afraid.

What am I doing wrong? 
Does 'autodiscover' has to be configured somewhere?
Is caldav / carddav even implemented in community version of Tine2.0?

Any suggestions please?
Gratefully Received
Thanks
Rob

**Reply by lab-at-nohl** [reply]
Have you tried to enter `https://yourdomain.com/webdav/` (*DAV) and `https://yourdomain.com/Microsoft-Server-ActiveSync` (Exchange) into your browser? 

If that does not asks for your username/password you are either missing **rewrite rules** or you have an improper setup... Look out hor the `.htaccess` file in docs.

**Reply by robz0** [reply]
Hi again Johannes, 
Thanks again for your fruitful suggestion which was progressive. 

I'll just share here:
There was a file in /docs/ but it was called htaccess (with not . in front). I'm not sure if that it how it was meant to be. I could see all the rewrite rules you mentioned there though.

So I renamed that file to ".htaccess".
Then I copied the '.htaccess' file to the root folder of the Tine2.0 installation.  It then worked as you expected - asking for a login when adding /webdav to the installation URL.

Going back to Thunderbird and Mac Calendar. It did not work as expected yet.  With a little more help from the web:
http://www.htaccess-guide.com/how-to-use-htaccess/
I changed permissions on the .htaccess file in the root directory to 755 and tried again. 
It worked on Thunderbird perfectly (obviously with the TBSync/Lighting addon)
- with nice and easy URL, thanks to the rewrites

Mac Calendar is a bit more complicated. 
It has been recognised with the following setup:
 'Advanced' configuration. 
SERVER - https://<the-site-url> 
 folder  - /tine/webdav
Port: 443 SSL - yes

but Currently is says: 
"The server specified an invalid calendar home."

I will keep experimenting with it though and feedback with any progress, if any, for the English speakers!

Thanks so much though for that. That was a perfect suggestion. 

Very Best Wishes once again.

Rob

**Reply by robz0** [reply]
Update: 
Mac OS native apps
Still no luck at all with Mac OS native apps - Calendar and Contacts, after trying various possibilities and combinations. 
I've also tried on two different macs running Calendar v9 and v10

Other apps - 
e.g. Thunderbird / Busy Contacts / Busy Calendar - work perfectly as described in the help docs for Tine2.0. 

So this is clearly an issue with Mac OS.  

Searching on the error - **"The server specified an invalid calendar home."** :
I see in some other, older forums (Own/NextCloud) that they have to provide a specific URL/link for mac os. to sync with caldav. Maybe there is something similar that has to be implemented. 

mobile:
More experimenting to do on mobile but Davx5 does recognise the calendars so far, though the actual apps (tasks/calendars/contacts) have yet to pick them up. 

Meanwhile, this is my Mac OS experience.
Thanks again for your guidance thus far.

**Reply by lab-at-nohl** [reply]
I am not an expert on Mac OS X since I don't use it myself. I guess the App just cannot find all calendars. Try if you can access `https://domain.com/principals/users/` in your browser. Please try to write a log file level DEBUG when accessing by Mac OS X CalDAV. 

If that all does not help, you can still click right on the calendar / Information. There you find the exact URL. Use this for your Calendar App. You won't have auto discovery but it works.

**Reply by msphn** [reply]
But next time maybe create an own discussion and don't add it to the welcome post
